core/src/main/java/brooklyn/entity/rebind/dto/MementosGenerators.java

```diff
@@ -127,13 +143,16 @@ public static BrooklynMemento newBrooklynMemento(ManagementContext managementCon
     
     /**
      * Inspects an entity to create a corresponding memento.
+     * <p>
+     * @deprecated since 0.7.0, see {@link #newBasicMemento(BrooklynObject)}
      */
+    @Deprecated
```

Comment:
Do we really need to keep all of these? The code is so specific that I am pretty sure no one know it exists. Also removing it won't mess with serialized state. I'd say delete it right now.